lib/SlackBot.js

```diff
@@ -338,7 +338,8 @@ function Slackbot(configuration) {
     slack_botkit.getAuthorizeURL = function(team_id, redirect_params) {
 
         var scopes = slack_botkit.config.scopes;
-        var url = 'https://slack.com/oauth/authorize' + '?client_id=' +
+        var api_host = process.env.api_host ? process.env.api_host : 'slack.com';
```

Comment:
I agree that it's more concise, but I think it's less readable, because the result is pretty unpredictable for js newbies. In many languages this statement will proceed boolean true, which will be converted to anything like 1, "true", etc.